### ISSUE TITLE
index.spt errors if no content-type in section 3

### ISSUE THREAD

**Issue by pjz** [gratipay/aspen-python#431]
if there is a single `index.spt` with content like:
```
[-----]
[-----]
Greetings, program!
```

and it is attempted to be accessed, it will produce:
```
pid-20164 thread-47544838462080 (MainThread) Traceback (most recent call last):
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/env/local/lib/python2.7/site-packages/algorithm.py", line 288, in run
pid-20164 thread-47544838462080 (MainThread)     new_state = function(**deps.as_kwargs)
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/aspen/algorithms/website.py", line 95, in get_resource_for_request
pid-20164 thread-47544838462080 (MainThread)     return {'resource': resources.get(website, dispatch_result.match)}
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/aspen/resources/__init__.py", line 176, in get
pid-20164 thread-47544838462080 (MainThread)     raise entry.exc[0]  # TODO Why [0] here, and not above?
pid-20164 thread-47544838462080 (MainThread) LoadError: Traceback (most recent call last):
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/aspen/resources/__init__.py", line 168, in get
pid-20164 thread-47544838462080 (MainThread)     entry.resource = load(website, fspath, mtime)
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/aspen/resources/__init__.py", line 137, in load
pid-20164 thread-47544838462080 (MainThread)     resource = Class(website, fspath, raw, media_type, is_bound, mtime)
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/aspen/resources/simplate.py", line 47, in __init__
pid-20164 thread-47544838462080 (MainThread)     self.pages = self.compile_pages(pages)
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/aspen/resources/simplate.py", line 129, in compile_pages
pid-20164 thread-47544838462080 (MainThread)     pages[2:] = (self.compile_page(page) for page in pages[2:])
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/aspen/resources/simplate.py", line 129, in <genexpr>
pid-20164 thread-47544838462080 (MainThread)     pages[2:] = (self.compile_page(page) for page in pages[2:])
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/aspen/resources/simplate.py", line 139, in compile_page
pid-20164 thread-47544838462080 (MainThread)     make_renderer, media_type = _parse_specline(page.header)
pid-20164 thread-47544838462080 (MainThread)   File "/home/pj/src/aspen-meta/aspen/aspen/resources/simplate.py", line 241, in _unbound_parse_specline
pid-20164 thread-47544838462080 (MainThread)     raise SyntaxError(msg)
pid-20164 thread-47544838462080 (MainThread) SyntaxError: Malformed media type  in specline . It must match [A-Za-z0-9.+*-]+/[A-Za-z0-9.+*-]+$.
pid-20164 thread-47544838462080 (MainThread)
```

Which seems to me to be `simplates.py` not paying attention to `self.media_type` even though it is correctly being set to `website.media_type_default`.

**Comment by pjz** [comment]
I put a failing test into the `issue431` branch.

**Comment by pjz** [comment]
Fixed by #433
